config/m4/verbs.m4

```diff
@@ -0,0 +1,317 @@
+# verbs.m4 - Parsing verbs capabilities
+# 
+# Copyright (C) Mellanox Technologies Ltd. 2016-2017.  ALL RIGHTS RESERVED.
+# See file LICENSE for terms.
+#
+
+##########################
+# Configure ofed capabilities
+#
+AC_DEFUN([VERBS_CAPABILITY_SETUP],
+[
+
+#
+# Experimental Verbs CQ
+#
+AC_ARG_ENABLE([exp-cq],
+    AC_HELP_STRING([--disable-exp-cq],
+                   [Disable experimental Verbs CQ (required for UDP RX HW Timestamp and for RX CSUM verification offload)]),
+    [],
+    [enable_exp_cq=yes]
+)
+
+AS_IF([test "x$enable_exp_cq" == xyes],
+        [AC_DEFINE([DEFINED_IBV_EXP_CQ], 1, [Define to 1 if Experimental Verbs CQ was enabled at configure time])]
+
+	AC_MSG_CHECKING([if IBV_EXP_CQ_TIMESTAMP is defined])
+	AC_TRY_LINK(
```

Comment:
all the AC_MSG,,AC_TRY should be replaced with one macro AC_CHECK_DECL or AC_CHECK_DECLS 
e.g.
AC_CHECK_DECL([IBV_EXP_CQ_TIMESTAMP],
[AC_DEFINE(DEFINED_IBV_EXP_CQ_TIMESTAMP, 1, [Define to 1 if IBV_EXP_CQ_TIMESTAMP is defined]],
 [], [[#include <infiniband/verbs_exp.h>]])